var/spack/repos/builtin/packages/ruby/package.py

```diff
@@ -41,6 +41,7 @@ class Ruby(Package):
     depends_on('openssl', when='+openssl')
     variant('readline', default=False, description="Enable Readline support")
     depends_on('readline', when='+readline')
+    depends_on('binutils', type='build')
```

Comment:
In the docs we recommend against adding binutils as a dependency:
http://spack.readthedocs.io/en/latest/getting_started.html#binutils